app/talk/quick-subject-comment-form.cjsx

```diff
@@ -19,12 +18,14 @@ PAGE_SIZE = talkConfig.discussionPageSize
 
 module.exports = React.createClass
   displayName: 'TalkQuickSubjectCommentForm'
-  mixins: [State, History]
```

Comment:
I couldn't see any of the State mixin methods used anywhere in this component...